ansible/group_vars/nimbus.holesky.yml

```diff
@@ -171,6 +176,13 @@ nimbus_era_files_network: '{{ beacon_node_network }}'
 nimbus_era_files_db_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/data/db'
 nimbus_era_files_nclidb_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/repo/build/ncli_db'
 
+# RPC-snooper
+rpc_snooper_service_name: 'rpc-snooper-{{ beacon_node_network }}-{{ "%02d"|format(idx|int+1) }}'
+rpc_snooper_cont_port:   '{{ 8700  + (idx|int) + 1 }}'
+rpc_snooper_el_port: '{{ exec_layer_authrpc_port }}'
+rpc_snooper_suppress_methods: []
+rpc_snooper_suppress_paths: []
```

Comment:
I don't see these being listed in `defaults/main.yml` of `infra-role-rpc-snooper`, at least commented out.